pkg/client/src/app/pages/reports/components/identified-risks-table/identified-risks-table.tsx

```diff
@@ -51,30 +60,27 @@ export const IdentifiedRisksTable: React.FC<
   // i18
   const { t } = useTranslation();
 
-  // Context
-  const { selectedItems: applications } = useContext(
-    ApplicationSelectionContext
-  );
-
-  // Toolbar filters data
   const {
-    filters: filtersValue,
-    isPresent: areFiltersPresent,
-    addFilter,
-    setFilter,
-    clearAllFilters,
-  } = useApplicationToolbarFilter();
+    allItems: allApplications,
+    selectedItems: selectedApplications,
+    areAllSelected: areAllApplicationsSelected,
+    isItemSelected: isApplicationSelected,
+    toggleItemSelected: toggleApplicationSelected,
+    selectAll: selectAllApplication,
+    setSelectedItems: setSelectedRows,
+    selectMultiple: selectMultipleApplications,
```

Comment:
None of the rest of these variables you pull out here are used, you may want to remove them again (or maybe it's useful to leave them here in case they are needed later?